packages/webex/src/encryption.js

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) 2015-2024 Cisco Systems, Inc. See the LICENSE file.
+ */
+
+// Note: this file is written using commonjs instead of import/export to
+// simplify consumption by those less familiar with the current state of
+// JavaScript modularization
+
+/* istanbul ignore else */
+if (!global._babelPolyfill) {
```

Comment:
webpack should automatically add it 

and not sure if we need to call this encryption as it includes decryption also , need to find a better name